templates/core/map.html

```diff
@@ -0,0 +1,91 @@
+{% load static %}
+
+<script>
+    mapboxgl.accessToken = 'pk.eyJ1IjoiZGphbmdvZ2lybHMiLCJhIjoiY2xjeWU1dm9iMG10cTNudGdrYTBkYm0ycSJ9.Ah4y76VVmBWqzVkw3kUzFQ'
```

Comment:
Thanks for all your work on this @aniav. The only concern I have is having the public key publicly displayed. All our keys are now environment variables. It would be great to have this as an environment variable as well.

Otherwise, great work!

Reply:
So is this about the key being public or about having keys configurable per environment? Because this key is intended to be publiczny visible. There is no way to pass a key through a JS script in a hidden way. Mapbox suggests configuring the domains that can use the key to make it more secure.

If it’s about making the key configurable I'll change that 🙂

Reply:
Yes, making it configurable would be great, thanks.